update.sh

```diff
@@ -49,6 +51,11 @@ for path in "${paths[@]}"; do
 
 	if [ -d "$path" ]; then
 	    cp docker-entrypoint.sh generate-jetty-start.sh "$path"
+	    if [ "$version" == "9.4" ] ; then
+	        echo '# DO NOT EDIT. Edit Dockerfile-9.4 and use update.sh' > "$path"/Dockerfile
+	    	cat Dockerfile-9.4 >> "$path"/Dockerfile
+	        sed -ri 's/^(FROM openjdk:)LABEL/\1'"$label"'/; ' "$path/Dockerfile"
+	    fi
 	    sed -ri 's/^(ENV JETTY_VERSION) .*/\1 '"$fullVersion"'/; ' "$path/Dockerfile"
```

Comment:
I wonder, would it be possible to change the "templating" of Dockerfile to use [build args](https://docs.docker.com/engine/reference/builder/#arg)?
The benefit would be having the Jetty full version changeable in a single place (Makefile, Travis config) without the need to modify all docker files for Jetty 9.4 for all JVM variants. Also, the version would be explicit, and there will be no dependency on parsing Maven repo metadata for latest string starting from `9.4`.

Reply:
I would think... But the practise appears to be to have all three variant dockerfiles checked in and a script to generate them if need be.

So I've kept it this way for now (plus I don't really know makefile well enough).  But if I can be told it is ok to parameterized and helped with the makefile....